tests/commands/autostep.py

```diff
@@ -19,7 +19,8 @@
 #
 import unittest
 from tests.baseclass import CommandTest
-from pykickstart.commands.autostep import FC3_AutoStep
+from pykickstart.commands.autostep import FC3_AutoStep, F34_AutoStep
```

Comment:
Importing `F34_AutoStep` isn't needed, and makes pylint warn about an unused import. Fix that and it looks fine to me.

Reply:
Thanks, done.